modules/core/shared/src/main/scala/gem/math/Angle.scala

```diff
@@ -227,17 +239,91 @@ object Angle extends AngleOptics {
 
 }
 
-trait AngleOptics { this: Angle.type =>
+trait AngleOptics extends OpticsHelpers { this: Angle.type =>
+
+  /**
+   * Microarcseconds, in [0, 360°).
+   * @group Optics
+   */
+  lazy val microarcseconds: SplitMono[Angle, Long] =
+    SplitMono(_.toMicroarcseconds, Angle.fromMicroarcseconds)
+
+  /**
+   * Signed microarcseconds, in [-180°, 180°).
+   * @group Optics
+   */
+  lazy val signedMicroarcseconds: SplitMono[Angle, Long] = {
+    lazy val µas360 = Angle.Angle180.toMicroarcseconds * 2L
+    microarcseconds.imapB[Long](
+      identity,
+      µas => if (µas >= Angle.Angle180.toMicroarcseconds) µas - µas360  else µas
+    )
+  }
+
+  /**
+   * Milliarcseconds, in [0, 360°).
+   * @group Optics
+   */
+  lazy val milliarcseconds: Wedge[Angle, Int] =
+    microarcseconds.scaled(1000L)
+
+  /**
+   * Arcseconds, in [0, 360°).
+   * @group Optics
+   */
+  lazy val arcseconds: Wedge[Angle, Int] =
+    microarcseconds.scaled(1000L * 1000L)
+
+  /**
+   * Arcminutes, in [0, 360°).
+   * @group Optics
+   */
+  lazy val arcminutes: Wedge[Angle, Int] =
+      microarcseconds.scaled(1000L * 1000L * 60L)
 
-  val hourAngle: Prism[Angle, HourAngle] =
-    Prism((_: Angle).toHourAngleExact)(_.toAngle)
+  /**
+   * Degrees, in [0, 360).
+   * @group Optics
+   */
+  lazy val degrees: Wedge[Angle, Int] = microarcseconds.scaled(1000L * 1000L * 60L * 60L)
```

Comment:
I don't yet have a feel for when you use `Wedge` instead of, I would guess, `SplitMono`.

Reply:
Going `Angle => Int` we squeeze `µas: Long` into `Int`, but on the way back we squeeze `Int` into `[0, 360)` so there is normalization happening in both directions.

The laws for all these new optics have a "coverage" test that checks to be sure you couldn't be using a stronger optic. For instance I accidentally had `Angle => DMS` as a `SplitEpi` and the test failed because it's actually an `Iso`. So when in doubt you can guess and the tests should tell you whether it's the right optic or not.